reference/OpenAPI.json

```diff
@@ -4195,7 +4195,7 @@
           }
         ]
       },
-      "VoucherQualificationValidationRedemption": {
+      "VoucherWithoutCategories": {
```

Comment:
Dlaczego "Without"?

Ma `categories` i to jeszcze poszerzone o `stacking_rules_type`.

Może: `VoucherWithStackingRulesTypeCategories`?